daprdocs/content/en/developing-applications/integrations/azure-api-management.md

```diff
@@ -0,0 +1,9 @@
+---
+type: docs
+title: "Dapr integration policies for Azure API Management"
+linkTitle: "Azure API Management"
+description: "Publish APIs for Dapr services and components through Azure API Management policies"
+weight: 3000
```

Comment:
```suggestion
weight: 5000
```

Reply:
@AaronCrawfis Should this be 6000 to put it below the existing "Cloud Providers" link instead? (Or should the Could Providers _index.md be bumped to 6000 if you want an explicitly ordered list?)

Reply:
Great point. We should actually place all the Azure ones under cloud providers folder. I'll merge this as is and PR a separate change for restructuring the folder